shm/Store.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "Alcohol.h"
+#include "Cargo.h"
+#include "Fruit.h"
+#include "Item.h"
+#include "Observer.h"
+#include "Player.h"
+#include "Time.h"
+
+enum class Response {
+    done,
+    lack_of_money,
+    lack_of_cargo,
+    lack_of_space
+};
+
+int min_cargo_amount = 0;
+int max_cargo_amount = 100;
+
+int min_price = 0;
+int max_price = 100;
+
+std::vector<ItemData> available_items = {
+    {"Excalibur", Rarity::legendary},
+    {"Pistol", Rarity::rare},
+    {"Barrel", Rarity::common},
+    {"Wooden Chest", Rarity::common},
+    {"Canon", Rarity::epic},
+    {"Eyepatch", Rarity::common},
+};
+
+std::vector<AlcoholData> available_alcohols = {
+    {"Pure alcohol", 96},
+    {"Rum", 30},
+    {"Ale", 8},
+    {"Beer", 3},
+};
+
+std::vector<FruitData> available_fruits = {
+    {"Banana", 10},
+    {"Apple", 5},
+    {"Watermelon", 3},
+};
```

Comment:
Since those three vectors are predetermined couldn't they be constexpr.

Reply:
+1